core/templates/dev/head/pages/exploration-editor-page/editor-navigation/editor-navigation-dropdown.directive.html

```diff
@@ -0,0 +1,151 @@
+<div class="dropdown hidden-md hidden-lg ng-cloak" style="position: absolute; right: 100px; top:10px;">
+  <a class="btn dropdown-toggle" style="background-color:#009b8c; color:white;" type="button" data-toggle="dropdown">Options
+    <span class="caret"></span>
+  </a>
+  <ul class="dropdown-menu" style="min-width: 100px !important;">
+    <li ng-if="showPublishButton()">
+      <div class="btn-group" style="margin-right: 10px; width:100%;"
+           uib-tooltip="<[getPublishExplorationButtonTooltip()]>"
+           tooltip-placement="bottom"
+           ng-class="{'disable-publish-button': isExplorationLockedForEditing() || countWarnings()}">
+        <button type="button" class="btn btn-default oppia-editor-publish-button protractor-test-publish-exploration" style="border:none; border-radius: 0px; padding-left:7px;"
+                ng-class="{'btn-success': !isExplorationLockedForEditing() && !countWarnings()}"
+                ng-click="showPublishExplorationModal()"
+                ng-disabled="isExplorationLockedForEditing() || countWarnings()">
+          <span ng-if="!publishIsInProcess">
+            <span>Publish</span>
+          </span>
+          <span ng-if="publishIsInProcess">
+            <i class="oppia-save-publish-loading oppia-save-publish-button-icon"
+               alt="Publish to Oppia Library">
+              <loading-dots ng-show="loadingDotsAreShown"></loading-dots>
+            </i>
+            <span class="oppia-save-publish-button-label">
+                  Publishing
```

Comment:
This should be indented by only 2 spaces from the previous line.

Reply:
I have removed the 2 spaces.